public/app/plugins/datasource/grafana-azure-monitor-datasource/components/NewMetricsQueryEditor/ResourceField.tsx

```diff
@@ -0,0 +1,152 @@
+import { css } from '@emotion/css';
```

Comment:
I originally copied this over when starting this ticket because I didn't know what changes it would require.

It should be lifted up to the the components directory. I think there was only 1 or 2 very minor changes required so it's definitely shareable.